scripts/index.js

```diff
@@ -0,0 +1,74 @@
+const setIndicatorColor = (tag, value) => {
+    if (value >= 80) {
+        tag.style.color = "red";
+    } else if ((value >= 70) && (value <= 79)) {
+        tag.style.color = "orange";
+    } else if ((value >= 60) && (value <= 69)) {
+        tag.style.color = "yellow";
+    } else if ((value >= 50) && (value <= 59)) {
+        tag.style.color = "green"; 
+    } else {
+        tag.style.color = "teal"
+    };
+};
+
+const setGroundLandscape = (tag, value) => {
+    if (value >= 80) {
+        tag.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+    } else if ((value >= 70) && (value <= 79)) {
```

Comment:
Minor note: the && is not necessary for this conditional test, we could just have:
```suggestion
    } else if (value >= 70) {
```